modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter *HTTP*.
+. In the *Operation* field, click *HTTP Listener*.
+. In the *Path* field, specify the path on which `HTTP Listener` listens for requests.
+. Click *Edit*:
++
+.Clicking *Edit* takes you to a screen where you can configure the HTTP Listener connection.
+image::introduction/intro-config-fd-exit.png[CLick Edit]
++
+. Click *Add Connection*.
+. In the *Connection Name* field, enter a connection name.
+. Select *Listener* as the connection type.
+. Enter values for the *Host* and *Port* fields:
+* To listen on a specific port for every network interface on the server, leave the default value `0.0.0.0` for *Host*.
+* To listen for incoming requests generated inside the server, set the *Port* value to `localhost`.
+. Optionally configure a reconnection strategy by expanding the *Advanced* tab and completing the fields in the *Connection* section.
++
+For example, this reconnection strategy does the following:
++
+* Forces the deployment to fail if the connection fails and configured reconnection attempts are exhausted
+* Limits the number of reconnection attempts to three at a frequency of every 5000 milliseconds (ms)
++
+.Use the fields in the Reconnection Strategy section to specify reconnection options
+image::introduction/intro-config-overview-reconnection-strategy-fd.png[Reconnection strategy example using Flow Designer]
+. To use TLS, scroll down and specify the required information in the *TLS Configuration* section.
+. Click *Save*.
+
+For more information about configuring `HTTP Listener`, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+[[scheduler]]
+=== Use Scheduler as an Input Source
+
+Use `Scheduler` to trigger a flow when a time-based condition is met. You can configure `Scheduler` to trigger a flow at a regular interval or give it a more flexible cron expression.
+
+To use `Scheduler` as an input source:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
++
+For example, the following configuration directs `Scheduler` to:
+
+* Wait 10 seconds before initiating the flow for the first time after the app starts
+* Initiate the flow every 30 seconds after the initial 10 second delay
++
+.Select `Fixed Frequency` in the *Scheduling Strategy* field to initiate a flow at regular intervals.
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to log `my message` every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
```

Comment:
For a less technical audience, you may want a quick sentence on what a Cron expression is.